src/app/sessions/book/page.tsx

```diff
@@ -1,16 +1,63 @@
 "use client";
 
+import { useRouter } from "next/navigation";
+import { IoArrowBackOutline } from "react-icons/io5";
+
 import { ExpandedSessionCard } from "@/components/booking/ExpandedSessionCard";
+import { Button } from "@/components/ui/button";
 import { useCartStore } from "@/stores/useCartStore";
 
+// import gameSessions from "@/types/game-session";
+
 export default function BookSessionPage() {
+  const router = useRouter();
+
+  const handleBackButtonClick = () => {
+    router.back();
+  };
+
+  const handleNextButtonClick = () => {
+    router.push("/sessions/book/payment");
+  };
+
   const cart = useCartStore((state) => state.cart);
+  const isPlayLevelSelected = cart.every(
+    (session) => session.playLevel !== undefined,
+  );
 
   return (
-    <div className="mx-5">
-      {cart.map((session) => (
-        <ExpandedSessionCard key={session.id} gameSession={session} />
-      ))}
+    <div className="flex flex-col h-dvh mx-4 gap-y-4">
+      <div className="flex mt-4 align-middle text-tertiary">
+        <Button
+          variant={"ghost"}
+          className="grid place-items-center mr-4 size-8"
+          size={"icon"}
+          onClick={handleBackButtonClick}
+        >
+          <IoArrowBackOutline size={24} />
+        </Button>
+        <span className="text-lg font-medium leading-none self-center">
+          Select your membership type
+        </span>
+      </div>
+
+      <div className="mx-5">
```

Comment:
can you remove this margin? This is stacked with the parent div margin and we only want 16px margins